### PR TITLE
fix(nd): font size and ts errors

### LINKED ISSUE #8087: [ND] Range Change and Mode Change messages overlapping with MAP NOT AVAIL
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-06-24T14:16:59+00:00",
    "ref": "refs/heads/master",
    "sha": "7c397d216b9deb6c0d0dc3e991a0ae7dd06b7d9b",
    "actor": "Benjozork",
    "event_name": "manual",
    "pretty_release_name": "master:7c397d21",
    "version": "v0.10.0-dev.7c397d2"
}
```

### Describe the bug

As visible on the videos, if the map is not available (IRS not aligned for e.g.) and the ND mode or range is changed, the associated message overlaps with the MAP NOT AVAIL message.

https://github.com/flybywiresim/a32nx/assets/119708186/d56d2745-9d54-4855-8e6f-2d88e606a30f

https://github.com/flybywiresim/a32nx/assets/119708186/dfd2569c-c465-4e7c-8802-a37dbdc768d7

### Expected behavior

Range Change and Mode Change messages not to appear if map is not available.

### Steps to reproduce

1) Change ND mode or range while the IRS are off or in alignment

### References (optional)

https://youtu.be/eugycw2-NTo?t=243

### Additional info (optional)

_No response_

### Discord Username (optional)

Bruno_pt99#5802

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #8087

## Summary of Changes

While finishing the NDv2, the font size of the approach message was changed to be more accurate. But other elements in the upper region on the ND were untouched. This adapts the font sizes of the remaining elements to match the approach message. 

Additionally, overlapping Range/Mode changes should be fixed not appear when MAP NOT AVAIL flag is being shown

There was also an error introduced in the PFD which is fixed now.

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1. Before aligning, switch ND modes and range and make sure they do not appear.
2. After aligning, do the same and ensure they do not overlap (range change message should override mode change message)
3. Make sure everything in the upper region of the ND is available and readable (GS/TAS/TO WPT Indication (distance/ETA...)
4. Turn off the FDs, ensure you are in selected heading and verify the heading indicator on the PFD shows up (given the selected heading is in range)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/instruments/src/PFD/PFD.tsx [hunk 1]
```diff
@@ -49,8 +49,6 @@ export class PFDComponent extends DisplayComponent<PFDProps> {
 
     private radioAltitudeFilter = new LagFilter(5);
 
-    private readonly headingWord = Arinc429Register.empty();
```
Comment (beheh) [inline fbw-a32nx/src/systems/instruments/src/PFD/PFD.tsx:52]: The import for Arinc429Register needs to be removed for CI to pass.

**Review by Benjozork — APPROVED** [review]

**Comment by tracernz** [comment]
Something to check for font sizes is compare PFD/ND from a square-on photo showing both to make sure there's a bit of consistency..

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #8063 
Tier of Testing : 1
Date : 25/6/2023

Testing Process:
- Start flight
- Check top ND, TO WPT, distance and ETA and runway indication, wind 


Negatives:
GS/TAS numbers are next to the labels without any margin, is this normal? If this is normal, consider this PASSED. If its not normal consider this NOT PASSED. 
image

Testing Results:
Unsure - Refer to negatives

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #8063 
Tier of Testing : 1
Date : 26/6/2023

Testing Process:
- Start flight
- Check top ND, TO WPT, distance and ETA and runway indication, wind 
- Check all ND pages

Testing Results:
Passed

**Pull request merged**